app/extensions/safe/components/wrapAddressBarButtonsRHS.js

```diff
@@ -94,10 +94,10 @@ const wrapAddressBarButtonsRHS = ( AddressBarButtons, extensionFunctionality = {
                     type="flex"
                     justify="space-between"
                     align="middle"
-                    className={ styles.toggleRow }
+                    className={ `${styles.toggleRow} ${CLASSES.SETTINGS_MENU__TOGGLE}` }
                 >
-                    <Col span={6} className={ styles.toggleText }>Toggle Experiments</Col>
-                    <Col span={6} offset={6}>
+                    <Col span={6} className={ `${styles.toggleText} ${CLASSES.SETTINGS_MENU__TOGGLE_TEXT}` }>Toggle Experiments</Col>
+                    <Col span={6} offset={6} classname={ `${styles.toggleText} ${CLASSES.SETTINGS_MENU__TOGGLE_BUTTON}` }>
```

Comment:
@shriram2301 two things here:

- This shouldn't have the `toggleText` class. It isn't text, and I'd wager that's not what the class is for at all. Please don't copy/paste indiscriminately, think about what you're adding.

- We should add the classes to the DOM element in question. Here you're adding them to the `Col` which is a layout tool. You should add your class to the `Switch` as that's what does the interaction (and if it were moved from without of this `Col`, it would then keep its class...)

Bonus point:

The toggle text classes are on a `Col`, it looks like as there's no wrapper on the text. Add a `span` around the text to achieve the same as the previous point above.

Reply:
I missed this one out.  Since we were only planning on testing the `history and `bookmarks` I left it as is. This was in the initial commit when I wanted to add class names. Thanks for pointing it out. I didn't revisit it as all we needed to do was check in the elements just existed. I've changed it with changes you requested and tested it.